api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionEmailAddress.java

```diff
@@ -67,12 +67,14 @@ public boolean equals(Object o) {
 
     DbInstitutionEmailAddress that = (DbInstitutionEmailAddress) o;
 
-    return Objects.equals(institution, that.institution)
-        && Objects.equals(emailAddress, that.emailAddress);
+    // don't refer to the institution itself - addresses can still be equal if unassigned or
```

Comment:
Can it? This object basically tracks an association between an email and an institution. The previous approach seems like it may have been more correct. If the issue is that we don't want to depend on the database's assigned ID during comparison, I'd just make that be the rule instead